notebooks/tutorials/1_getting_started_lesson2/notebook.ipynb

```diff
@@ -0,0 +1,173 @@
+   "source": [
+    "Research provides utility functions to query pricing, volume, and returns data for 8000+ US equities, from 2002 up to the most recently completed trading day. These functions take an asset (or list of assets) along with a start and end date, and return a pandas [Series](http://pandas.pydata.org/pandas-docs/version/0.18/generated/pandas.Series.html) (or [DataFrame](http://pandas.pydata.org/pandas-docs/version/0.18/generated/pandas.DataFrame.html)) indexed by date.  \n",
+    "\n",
+    "Let's define the period of time we want to explore and use the `returns` function to query data for AAPL:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "2014-01-02 00:00:00+00:00   -0.014137\n",
+       "2014-01-03 00:00:00+00:00   -0.022027\n",
+       "2014-01-06 00:00:00+00:00    0.005376\n",
+       "2014-01-07 00:00:00+00:00   -0.007200\n",
+       "2014-01-08 00:00:00+00:00    0.006406\n",
+       "2014-01-09 00:00:00+00:00   -0.012861\n",
+       "2014-01-10 00:00:00+00:00   -0.006674\n",
+       "2014-01-13 00:00:00+00:00    0.005043\n",
+       "2014-01-14 00:00:00+00:00    0.020123\n",
+       "2014-01-15 00:00:00+00:00    0.020079\n",
+       "Freq: C, Name: Equity(24 [AAPL]), dtype: float64"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Research environment functions\n",
+    "from quantopian.research import returns, symbols\n",
+    "\n",
+    "# Select a time range to inspect\n",
+    "period_start = '2014-01-01'\n",
+    "period_end = '2014-12-31'\n",
+    "\n",
+    "# Query returns data for AAPL\n",
+    "# over the selected time range\n",
+    "aapl_returns = returns(\n",
+    "    assets=symbols('AAPL'),\n",
+    "    start=period_start,\n",
+    "    end=period_end,\n",
+    ")\n",
+    "\n",
+    "# Display first 10 rows\n",
+    "aapl_returns.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Alternative Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to pricing and volume data, Quantopian integrates a number of alternative datasets that include corporate fundamentals, stock sentiment analysis, and macroeconomic indicators to name a few. You can find the complete list of 50+ datasets on Quantopian's [data page](https://www.quantopian.com/data).  \n",
+    "\n",
+    "Our goal in this tutorial will be to build an algorithm that selects and trades assets based on sentiment data, so let's take a look at PsychSignal's [StockTwits Trader Mood](https://www.quantopian.com/data/psychsignal/stocktwits) dataset. This dataset assigns bull and bear scores to stocks each day based on the aggregate sentiment from messages posted on Stocktwits, a financial communications platform.  \n",
```

Comment:
I would make `bull` and `bear` hyperlinks to bull markets and bear markets, just to be clear what the dataset is trying to say.

Reply:
Or explain it in-line.